ddtrace/tracer/util_test.go

```diff
@@ -79,7 +79,6 @@ func TestIsValidPropagatableTraceTag(t *testing.T) {
 	}{
 		{"hello", "world", nil},
 		{"hello=", "world", fmt.Errorf("key contains an invalid character 61")},
-		{"hello", "world=", fmt.Errorf("value contains an invalid character 61")},
```

Comment:
Maybe just change this to be nil instead of an error, since you expect this to be valid, right? No need to delete the test case entirely.